install-unit

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-set -e # Exit if any command fails
+set -ex # Exit if any command fails
```

Comment:
tell me what `set -e` does, and i don't think you should include `set -x` it makes the output crazy verbose, which is helpful for debugging, but adds a lot of noise to an install that is working correctly